### ISSUE TITLE
[Survival Beta - Suggestion] Early Stages

### ISSUE THREAD

**Issue by Xerma** [tsunamayo/Starship-EVO#5482]
I really do like how it feels so far but the main thing I think is an issue early on is having all these resources that you need to find and bring back to your base. The amount of different ores feels fine, but having to walk around a lot to hopefully find isn't exactly fun.

Perhaps a good early intro/tutorial could be a ship that is far more wrecked/crashed with some storage containers and a busted hover bike that you have to repair or just make work. That way new players get a sense of building but then they also have a smoother way to find these ores

**Comment by tsunamayo** [comment]
Yes I agree it get quite tedious until you get your first hovercraft. I am not sure on the best way to address this, this is why I spawn a wreck (so you get iron) and I added a boost to walk faster.

**Comment by Xerma** [comment]
Yeah, getting that proper balance will of course take some time. Not sure if you want me to make a separate post for this but the early parts almost feel pointless right now considering I got an arc furnace and assembler going in under an hour. It's late for me right now so my brain isn't at full force.

I'm unsure if the later stage items just need to be more expensive to make, but the early stuff needs a way to still be useful for longer. They almost need a special item or an ore that is rarer. That would give a higher purpose of wanting to create an early transport.

The arc furnace especially should have something to set it back more. It may even be that windmills are far too strong seeing as I only have 1 windmill and it powers an assembler and arc furnace easily. 

Overall, the first crafting bench and furnace become pointless after just about an hour, which seems very fast

**Comment by SovietSponge** [comment]
I also had the materials to make a functional ship almost immediately whereas I was missing some ore to even make hover pods. The whole making a hovercraft first to get around is just kind of unnecessary with how easy it is to just make a ship instead.

**Comment by tsunamayo** [comment]
Yes I will have to make hovercraft easier, and have the player to travel a bit for the spaceship.
I can make the arc furnace harder to get, also I wanted to make a post on this but the power production ladder will have to be redone, I think wind is too powerful.

@SovietSponge for spaceship you need to get some hydrogen, which is a bit more complex. For hovercraft I want to add swappable battery cell so it is easier.

**Comment by SovietSponge** [comment]
@tsunamayo Making hovercraft easier is a solid step but making fuel is also a very easy hurdle to overcome, I found actually building a ship took me longer than making the fuel I needed to achieve a 200 minute runtime.
